### DISCUSSION TITLE
lex_worker??

### DISCUSSION THREAD

**Discussion by LCLabQU** [l3atbc-datadog/pushkin#135] [General]
Hi, 

following Tutorial: Simple experiment

pushkin start;

gets hung up at the following place:

.
.
.

Pulling lex_worker (lex_worker:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]N

I've tried responding y and N. In either case, the process stalls. 

Any advice?
Thanks!

**Comment by parkerrobbins** [comment]
Hi, @LCLabQU! Check out the solutions proposed in #100. Please let me know if that helps!

**Reply by LCLabQU** [reply]
No, this doesn't help. 
I installed pushkin 2.0, so there were multiple kill today. I haven't done armageddon though. I also tried rm -rf experiments/.DS_Store. 
Still getting the error...

**Reply by parkerrobbins** [reply]
Just to confirm, you have run `pushkin prep` again before running `pushkin start;`? 

If you have, what operating system are you using (with version)? We can try to reproduce your error. Thanks!

**Reply by LCLabQU** [reply]
Yes, I did re-run all the commands (and double checked /re-run all after reading #100). I'm on Mac OS 10.15.5

**Reply by parkerrobbins** [reply]
Okay, could you post the output from (1) `pushkin prep` and (2) as far as you get in `pushkin start;`? Thanks!

**Reply by parkerrobbins** [reply]
Hello, @LCLabQU! Just checking in to see if you ever got this to work. If not, let us know and we'd be happy to help further. You might also follow the steps outlined in #138.

**Reply by parkerrobbins** [reply]
Hey, @LCLabQU! If you're still having problems, we're here and happy to help. As a first step, you can follow the instructions [here](https://github.com/pushkin-consortium/pushkin/discussions/138#discussioncomment-58671) to upgrade to the latest version of the CLI. Please let us know if there's anything we can do!

**Reply by parkerrobbins** [reply]
Hey, @LCLabQU! Hope all is well. If you have a moment, we'd love to know if your issue is resolved or if you still need help!
